cli/args/flags.rs

```diff
@@ -2951,13 +2953,27 @@ Evaluate a task from string
           .help("Specify the directory to run the task in")
           .value_hint(ValueHint::DirPath),
       )
+      .arg(
+        Arg::new("recursive")
+          .long("recursive")
+          .short('r')
+          .help("Run the task in all projects in the workspace")
+          .action(ArgAction::SetTrue),
+      )
+      .arg(
+        Arg::new("filter")
+        .long("filter")
+        .short('f')
+        .help("Filter members of the workspace by name - should be used with --recursive")
```

Comment:
```suggestion
        .help("Filter members of the workspace by name - should be used with --recursive")
        .requires("recursive")
```

Reply:
actually, maybe we should make `--filter` imply recursive

Reply:
This broke the tests so I reverted it. We should resolve this before landing.

Reply:
Changed it to make `--filter` imply `--recursive`.